src/Microsoft.HttpRepl/StringResources.cs

```diff
@@ -0,0 +1,7 @@
+namespace Microsoft.HttpRepl
+{
+    internal static class StringResources
+    {
+        public static readonly string Error_NoBasePath = "'set base {url}' must be called before issuing requests to a relative path";
```

Comment:
This needs to be localized in future. Should we put it in a Strings.Resx file instead?

Reply:
Done in e3302fb.